source/boards/work-with-cards.rst

```diff
@@ -22,3 +22,53 @@ Cards can have different configuration fields, depending on the type of board yo
 
+Mention people
+--------------
+
+You can include a team member on a card by mentioning them on a card <https://docs.mattermost.com/messaging/mentioning-teammates.html>__ the same way you would in Channels. The team member you mention will receive a Direct Message notification from the Boards bot with a link to the card you mentioned them on. To mention multiple team members, separate each name with a comma.
+
+Mention auto-completes will be available in a future release.
+
+Share card previews
+-------------------
+
+When you share a link to a card within Mattermost, the card details are automatically displayed in a preview. This preview highlights what the card is about at a glance without having to click through to it.
+
+Customize card properties
+-------------------------
+
+Cards can contain different data fields depending on the purpose of the board. For example, in a Roadmap board, cards include a Type field where you can add categories such as Bug, Triage, etc. In a Task board, cards include Status and Priority fields instead.
+
+Add and manage properties
+-------------------------
+
+On a card, you can create a new property field by selecting Add a property, then selecting the type of property from the drop-down menu. The property type specifies the type of data you plan to capture within that field. When you create new card properties, they are added to all new and all existing cards on the current board.
+
+Properties are automatically added to the board filter list at the top of the page, so ensure you customize all property names to make it easy to filter your board by specific properties later.
+
+You can rename a property at any time by opening a card, then selecting New Property. Your changes are saved immediate, and applied across all cards on the current board.
+
+You can also delete properties you no longer need by selecting the property, then choosing Delete.  You'll be asked to confirm that you want to removes that property from every card on the current board.
+
+Once you have card properties defined, you have full control over which properties are visible on the board. Click Properties at the top of the board, then enable all properties you want to see at a glance, and hide all properties you don't want to see.
+
+Work with property types
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+* **Checkboxes** are useful for agenda items that are regularly revisited in weekly sprint or grooming meetings.
+* **Multi-select** allows you to create badges to indicate things like status.
+* **Numbers** are useful to capture metrics such as task sizing or effort estimates.
+* **Person** provides a quick way to capture user assignments.
+
+Once you've added a property and named it, to display this information in the view you're using, select **Properties** and then select the property name.
+
+Calculations
```

Comment:
@wuwinson: Some questions around calculations that I'm wondering if you can help with:

- What properties and types do you need on a card to be able to take advantage of board calculation options?
- What real-world examples do we have to demonstrate how calculations can be useful (based on the board's focus) other than the ones below? 

Thank you!

Reply:
@justinegeffen Played around with the calculations tool and documented my findings below. @harshilsharma63 @ogi-m @chenilim Can you please confirm if this is accurate:

1. **Count**: Counts the total number of rows in Table view or total number of cards in a column in Board view. Applies to any property type.
2. **Count Empty:** Applies to any property type
    1. Table View: Counts the total number of empty rows per column selected
    2. Board View: Counts the total number of empty values per property specified within the same column
3. **Count Not Empty**: Applies to any property type
    1. Table View: Counts the total number of rows with non-empty cells per column selected
    2. Board View: Counts the total number of non-empty values per property specified within the same column
4. **Percent Empty**: Applies to any property type
    1. Table View: Percentage of empty rows per column selected
    2. Board View: Percentage of empty values per property specified within the same column
5. **Percent Not Empty**: Applies to any property type
    1. Table View: Percentage of rows with non-empty cells per column selected
    2. Board View: Percentage of non-empty values per property specified within the same column
6. **Count Value**: Applies to any property type
    1. Table View: Counts the total number of values within the column (helpful for multi-select properties)
    2. Board View: Counts the total number of values per property specified within the same column.
7. **Count Unique Values**: Applies to any property type
    1. Table View: Counts the total number of rows with unique values within the column, omitting any duplicates from the count.
    2. Board View: Counts the total number of unique values per property specified within the same column, omitting any duplicates from the count.
8. **Sum**: The sum of any specified number property within the same column
9. **Average**: The average of any specified ****number property within the same column
10. **Median**: The median of any specified number property within the same column
11. **Min**: The lowest number of any specified number property within the same column
12. **Max**: The highest number of any specified number property within the same column
13. **Range**: Displays the lowest and highest number. Requires a number property.
14. **Earliest Date**: Displays the oldest date. Requires any custom date property or the included "Created time" or "Last updated time"
15. **Latest Date**: Displays the most recent date. Requires any custom date property or the included "Created time" or "Last updated time"
16. **Date Range**: The difference between the most recent date and oldest date within the same column. In Table View, it is labeled simply as "Range" for any date property/column. Requires any custom date property or the included "Created time" or "Last updated time"


Below are some other examples Chen provided in the v6.1 blog post on how calculations can be used:
- The count of cards where particular properties are empty (useful to make sure important info isn’t missing)
- The sum of estimated developer days for features (to make sure your team isn’t overloaded), and
- The range of estimated dates (to make sure your milestones all line up)

Reply:
@wuwinson that's absolutely correct! Sorry for missing this notification.